src/main/java/com/softserveinc/ita/pageobjects/admin/GroupsPage.java

```diff
@@ -1,12 +1,15 @@
 package com.softserveinc.ita.pageobjects.admin;
 
 import com.softserveinc.ita.pageobjects.modals.AddingFormModal;
+import io.qameta.allure.Step;
 
 import static com.codeborne.selenide.Selenide.$x;
 
 public class GroupsPage extends MainMenu {
+    @Step("Groups page: opened adding groups form")
     public AddingFormModal openAddingForm() {
```

Comment:
I see that this method is very similar in different page objects, maybe it makes sense to make it in MainMenu?